.github/workflows/e2e-testing.yml

```
@@ -13,12 +14,28 @@ jobs:
         with:
           node-version: ${{ vars.NODE_VERSION }}
 
+   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)